docs/devops.md

```diff
@@ -34,11 +34,12 @@ In 2012, Puppet Labs “state of DevOps report” benchmarked 4,039 IT orgs usin
 
 ***
 
->“*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”*   
-**-Phoenix Project**
+> “*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”* ** - Phoenix Project **
 
 ***
 
+![](https://user-images.githubusercontent.com/5632966/105532539-95abdd00-5cb0-11eb-81ff-b854c4ec1784.png)
```

Comment:
Around the web page we have images with and without description/name, i think is better to add a short name for the image.

Reply:
Nice!! 🥇

Reply:
I'm going to remove this link, later in the same docs is use the same one in different context.